.github/workflows/code-style.yml

```diff
@@ -1,18 +1,18 @@
 name: Format python code
 on: push
 jobs:
-  autopep8-black:
+  code-style:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@master
-    - name: autopep8
-      uses: peter-evans/autopep8@v1.0.0
+    - uses: actions/setup-python@v1
       with:
-        args: --recursive --in-place --aggressive --aggressive .
-    - name: autoblack
-      uses: lgeiger/black-action@v1.0.1
-      with:
-        args: "."
+        python-version: '3.7'
+    - run: |
+        python -m pip install autopep8 black isort
+        isort -r .
```

Comment:
`-r` here stands for recursive ?

Reply:
You're right, fa268bbab25649bd9f6d15e5ba84a1a6ec9bbc0e fixes it.